package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vets-json-schema",
-  "version": "6.6.3",
+  "version": "6.7.3",
```

```suggestion
  "version": "6.7.0",
```

Per [semver](https://semver.org), shouldn't this be `6.7.0`?

---

Right, sorry we missed that!